src/svg-pan-zoom.js

```diff
@@ -137,16 +143,34 @@ SvgPanZoom.prototype.setupHandlers = function() {
 
     // Mouse leave group
   , mouseleave: function(evt) {
+      if (!that.options.triggerUpOnLeave) {
+        return;
+      }
       return that.handleMouseUp(evt);
     }
   , touchleave: function(evt) {
+      if (!that.options.triggerUpOnLeave) {
+        return;
+      }
       return that.handleMouseUp(evt);
     }
   , touchcancel: function(evt) {
       return that.handleMouseUp(evt);
     }
   }
 
+  if (that.options.useGlobalMove) {
+    delete that.eventListeners.mousemove;
+    delete that.eventListeners.touchmove;
+    that.globalMoveHandler = function(evt) {
+      if (that.options.restrictPanWhilePointerIsDown && !that.isDown) {
+        return;
+      }
+      return that.handleMouseMove(evt);
+    };
+    window.addEventListener('pointermove', that.globalMoveHandler);
```

Comment:
Does it work in IE9? [MDN docs mention](https://developer.mozilla.org/en-US/docs/Web/Events/pointermove) IE10+. If it doesn't work with IE9 and we definitely want to move away from it - then we'll have to create a new major version that would not support IE9 anymore.